extensions/2.0/Vendor/EXT_mesh_primitive_restart/schema/primitiveGroup.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "$id": "primitiveGroup",
+    "title": "Primitive Group",
+    "type": "object",
+    "description": "A group of primitives that can be drawn together using a single index buffer containing restart values",
+    "allOf": [
+        {
+            "$ref": "glTFProperty.schema.json"
+        }
+    ],
+    "properties": {
+        "primitives": {
+            "type": "array",
+            "items": {
+                "$ref": "glTFid.schema.json"
+            },
+            "uniqueItems": true,
+            "minItems": 1,
+            "description": "The indices of the primitives that will be combined and drawn using a single set of indices",
+            "gltf_detailedDescription": "The indices of the primitives that will be combined and drawn using a single set of indices. All properties except for `indices` (e.g., `material`, `mode`, and `attributes`) will be obtained from the first primitive in the list, and all other primitives in the list **MUST** have identical values for those properties. The `mode` must be TRIANGLE_FAN, TRIANGLE_STRIP, LINE_LOOP, or LINE_STRIP."
+        },
+        "indices": {
+            "allOf": [
+                {
+                    "$ref": "glTFid.schema.json"
+                }
+            ],
+            "description": "The index of the accessor that contains the vertex indices",
+            "gltf_detailedDescription": "The index of the accessor that contains the vertex indices. The accessor **MUST** have `SCALAR` type and an unsigned integer component type. The indices are permitted to include the maximal index value for the component type, indicating the start of a new primitive."
+        },
+        "name": {},
+        "extensions": {},
+        "extras": {}
+    },
+    "required": [
+        "primitives",
+        "indices"
+    ]
+}
```

Comment:
Nitpick: The file should have a trailing `\n` to be a POSIX-compliant text file. GitHub displays a ⛔ symbol on such files.